practitioner/src/main/java/org/smartregister/extension/rest/PractitionerDetailsResourceProvider.java

```diff
@@ -450,6 +466,20 @@ private List<String> getLocationIdentifiersByIds(List<Long> locationIds) {
         return locationsIdentifiers;
     }
 
+    private List<IBaseResource> getP(Long practitionerId) {
```

Comment:
Please name the variables better. Variables show be self-documenting e.g `getParticipant`